apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -862,15 +862,15 @@ public void OnlyOneStart()
             }
         }
 
-        [Fact]
-        public void LeaseExtension()
+        [Theory, CombinatorialData]
+        public void LeaseExtension([CombinatorialValues(true, false)] bool isExactlyOnceDelivery)
```

Comment:
You don't need this, the test is already marked as CombinatorialData, so bools and I believe enums, etc, it will combine the values automatically.

```suggestion
        public void LeaseExtension(bool isExactlyOnceDelivery)
```

Reply:
Done.